lib/active_record/virtual_attributes/virtual_fields.rb

```diff
@@ -363,7 +328,9 @@ def apply_join_dependency(*args, **kargs, &block)
       # From ActiveRecord::QueryMethods (rails 5.2 - 6.0)
       def build_select(arel)
         if select_values.any?
+          # change (only place allow_alias is true - the select clause)
```

Comment:
I'm not sure what this means: `(only place allow_alias is true - the select clause)`... maybe say why there is a change here?  I can't figure it out at a glance.